### ISSUE TITLE
Nonstandard properties tooltips content

### ISSUE THREAD

**Issue by taylornowotny** [webstudio-is/webstudio-builder#2124]
in progress

| nonstandard prop | custom tooltip description
|-|-|
| Row 1, Col 1   | Row 1, Col 2   
| Row 2, Col 1   | Row 2, Col 2  
| Row 3, Col 1   | Row 3, Col 2

**Comment by taylornowotny** [comment]
Original list of all properties provided by @istarkov 

sdk-components
```
Box
  tag
-------------------------------
CodeText
  meta
  inline
-------------------------------
Heading
  tag
-------------------------------
HtmlEmbed
  code
  executeScriptOnCanvas
-------------------------------
Image
  quality
  optimize
-------------------------------
Link
  target
-------------------------------
List
  ordered
-------------------------------
RichTextLink
  target
-------------------------------
Text
  tag
-------------------------------
VimeoPreviewImage
  quality
  optimize
-------------------------------
Vimeo
  url
  autopause
  autopip
  autoplay
  keyboard
  loop
  muted
  pip
  playsinline
  quality
  responsive
  speed
  texttrack
  transparent
  showPreview
  doNotTrack
  interactiveParams
  backgroundMode
  showControls
  controlsColor
  showByline
  showTitle
  showPortrait
```

**Comment by taylornowotny** [comment]
Radix: per component
```-------------------------------
Accordion
  dir
  disabled
  orientation
  value
AccordionItem
  value
  disabled
AccordionHeader
AccordionTrigger
AccordionContent
  forceMount
-------------------------------
Button
  variant
  size
-------------------------------
Collapsible
  open
  disabled
CollapsibleTrigger
CollapsibleContent
  forceMount
-------------------------------
Dialog
  open
  modal
DialogTrigger
DialogOverlay
  asChild
  forceMount
DialogContent
  forceMount
  asChild
DialogClose
  asChild
DialogTitle
  asChild
DialogDescription
  asChild
-------------------------------
Input
-------------------------------
Label
-------------------------------
Popover
  open
  modal
PopoverTrigger
PopoverContent
  asChild
  side
  sideOffset
  align
  alignOffset
  arrowPadding
  collisionBoundary
  collisionPadding
  sticky
  hideWhenDetached
  avoidCollisions
  forceMount
-------------------------------
Sheet
  open
  modal
SheetTrigger
SheetOverlay
  asChild
  forceMount
SheetClose
  asChild
SheetTitle
  asChild
SheetDescription
  asChild
SheetContent
  asChild
  forceMount
  tag
  side
-------------------------------
Tabs
  dir
  value
  orientation
  activationMode
TabsList
  loop
  asChild
TabsTrigger
  value
TabsContent
  value
  forceMount
-------------------------------
Textarea
-------------------------------
Tooltip
  open
  delayDuration
  disableHoverableContent
TooltipTrigger
TooltipContent
  aria-label
  asChild
  side
  sideOffset
  align
  alignOffset
  arrowPadding
  collisionBoundary
  collisionPadding
  sticky
  hideWhenDetached
  avoidCollisions
  forceMount```

**Comment by ggurgone** [comment]
FYI here is the PR to add tooltips for component props. Right it only supports HTML attributes https://github.com/webstudio-is/webstudio-builder/pull/2125

**Comment by taylornowotny** [comment]
Thanks, I'll use this issue for discussions around nonstandard props then add them to #2125 when content is ready.

**Comment by taylornowotny** [comment]
Noted: asChild and forceMount will not need tooltips content

**Comment by taylornowotny** [comment]
Noted: @istarkov confirmed that he intends to remove the "Modal" property from all components. Please comment if I am mistaken about this.

**Comment by taylornowotny** [comment]
closed via #2125
